store.go

```diff
@@ -2088,12 +2076,11 @@ func (s *store) ContainerSize(id string) (int64, error) {
 		return -1, err
 	}
 
-	var res int64 = -1
-	err = s.writeToContainerStore(func() error { // Yes, s.containerStore.BigDataSize requires a write lock.
+	return writeToContainerStore(s, func() (int64, error) { // Yes, rcstore.BigDataSize requires a write lock.
```

Comment:
Where did `rcstore` come from?

Reply:
Thanks, fixed. (It’s a mistake in rebasing, `rcstore` used to exist back when this branch was originally prepared.)